tracing-api/src/main/java/com/palantir/tracing/api/TraceTags.java

```diff
@@ -70,8 +70,8 @@
     // The data would likely be unsafe to share anyhow.
 
     // 'http.url_details.host' may contain unsafe data and is currently excluded out of caution.
```

Comment:
I wrote the comment, but neglected to comment the value. I like listing the constants in this file to match the upstream documentation rather than removing them entirely, that way it's obvious why certain constants aren't listed.